modules/ctrl_dbus/ctrl_dbus.c

```diff
@@ -137,9 +137,10 @@ static void command_handler(int flags, void *arg)
 	mbuf_set_pos(st->mb, 0);
 	st->command = mem_deref(st->command);
 
+out:
 	pthread_mutex_lock(&st->wait.mutex);
 	pthread_cond_signal(&st->wait.cond);
-	read(st->fd[0], buf, sizeof(buf));
+	(void)read(st->fd[0], buf, sizeof(buf));
```

Comment:
if read fails, the content of `buf` will be uninitialized.

it is probably good to init the buffer and to check the return value of read.